pkg/agent/agent.go

```diff
@@ -20,14 +20,15 @@ import (
 
 // Agent configures the cluster agent
 type Agent struct {
-	APIURL       *url.URL
-	Token        string
-	ClusterID    string
-	CloudType    string
-	CloudRegion  string
-	Distribution string
-	Namespace    string
-	ArgoCDImage  string
+	APIURL           *url.URL
+	Token            string
+	ClusterID        string
+	CloudType        string
+	CloudRegion      string
+	Distribution     string
+	Namespace        string
+	ArgoCDImage      string
+	ArgoCDRedisImage string
```

Comment:
I understand the reason why you'd go for that name, but it's also unlikely that we add more different Redises anyway
```suggestion
	RedisImage string
```
(suggestion also applies to the rest of the code, like flags, etc.)